dart/lib/src/protocol/dsn.dart

```diff
@@ -43,7 +43,7 @@ class Dsn {
       apiPath = 'api';
     }
     return Uri.parse(
-      '${uriCopy.scheme}://${uriCopy.host}$port/$apiPath/$projectId/store/',
+      '${uriCopy.scheme}://${uriCopy.host}$port/$apiPath/$projectId/envelope/',
```

Comment:
envelope endpoint requires `Content-Type=application/x-sentry-envelope`  iirc, I dont see this in this PR

Reply:
https://develop.sentry.dev/sdk/envelopes/#http-headers